pkg/lca/imagebasedupgrade.go

```diff
@@ -273,6 +273,21 @@ func (builder *ImageBasedUpgradeBuilder) WithOadpContent(
 	return builder
 }
 
+// AutoRollbackOnFailureDisableForPostReboot allows controlling
+// AutoRollback on failure for post reboot stage. Enabled by default.
+func (builder *ImageBasedUpgradeBuilder) AutoRollbackOnFailureDisableForPostReboot(
+	flag bool) *ImageBasedUpgradeBuilder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Setting Auto Rollback on failure for post reboot to %b in imagebasedupgrade", !flag)
```

Comment:
Please update log to>
Setting Auto Rollback on failure flag for post reboot to %b in imagebasedupgrade

Reply:
there's an inverse logic - note that I'm inversing the flag's value.
"disableautorolback" it's disabled by default, which means it's enabled